examples/modules/d.py

```diff
@@ -0,0 +1,43 @@
+import asyncio
+
+from magda.module import Module
+from magda.decorators import register, accept, finalize, produce
+
+from examples.modules.common import log
+from examples.interfaces.common import Context
+from examples.interfaces.fn import LambdaInterface
+
+
+@accept(LambdaInterface)
+@produce(LambdaInterface)
+@register('D')
+@finalize
+class ModuleD(Module.Runtime):
+    SLEEP_TIME = 1
+
+    def bootstrap(self):
+        ctx: Context = self.context
+        log(self, ctx.timer, '--- Created!')
+        log(self, ctx.timer, self.parameters['threshold'])
```

Comment:
Make this log more verbose, e.g.
```suggestion
        log(self, ctx.timer, f"Threshold = {self.parameters['threshold']}")
```

Reply:
Good point